src/util.rs

```diff
@@ -72,13 +72,16 @@ impl Case {
         }
     }
     pub fn sanitize<'a>(&self, s: &'a str) -> Cow<'a, str> {
-        let s = sanitize(s);
+        self.cow_to_case(s.into())
+    }
```

Comment:
doesn't this make sanitize not sanitize anymore?

Reply:
yes :+1:  did not see that, I just did a quickfix here because I received foldernames like espressifesp32